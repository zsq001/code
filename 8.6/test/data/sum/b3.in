100 100
2384776 2894802 3238325 2819681 2876298 1294095 63898 2597373 636690 917656 1455476 1004888 1176522 979615 535993 2570381 2606699 654327 1875674 1153733 2286510 3108397 470648 265921 592488 105983 406120 1280843 1775614 2754537 1347349 1644889 2173270 853863 1449596 168182 1756401 1089092 1125643 2845114 2868987 182662 262891 2627614 3040106 977079 2687000 2293130 1169130 2929966 1262525 1860534 766906 2410136 2812473 2494126 682446 601043 709959 1267962 2546604 1282015 821163 2450250 2260823 3010448 1543911 53907 1413235 1270581 622315 2924072 2905163 881240 2521757 1797794 388085 2783708 2871478 1838373 865597 730522 140359 155728 1785730 1733572 501768 3013613 1264600 674961 682397 671394 2449892 1419650 436768 1802561 1503436 2312691 2411261 2872442 
1 41 92 1473416
2 3 51
1 35 56 3275230
2 58 73
1 24 84 2650559
2 12 86
1 67 86 2393911
2 17 97
1 27 28 971487
2 27 29
1 12 96 358981
2 18 51
1 15 64 34748
2 1 90
1 72 77 536234
2 18 97
1 24 57 881171
2 70 80
1 66 67 411892
2 8 93
1 13 27 909389
2 61 74
1 44 73 3119964
2 57 72
1 11 54 1439918
2 54 93
1 22 58 693552
2 42 94
1 28 86 2970405
2 10 26
1 3 70 333137
2 43 97
1 15 72 901172
2 40 66
1 18 51 2737465
2 64 66
1 46 52 2750175
2 48 56
1 22 86 3230321
2 96 100
1 24 65 1878228
2 35 50
1 4 60 587905
2 70 85
1 27 32 308350
2 19 76
1 6 57 2512292
2 89 99
1 62 99 1996002
2 7 61
1 24 87 2069618
2 94 94
1 6 66 1632706
2 18 70
1 49 81 55167
2 35 85
1 34 56 864972
2 16 87
1 11 88 1303834
2 21 28
1 29 90 1449326
2 14 66
1 7 20 1088562
2 50 70
1 29 33 1489748
2 82 88
1 2 30 806360
2 17 36
1 8 81 2540533
2 38 42
1 65 94 578613
2 19 20
1 14 16 1718766
2 1 12
1 22 91 1279418
2 10 72
1 12 15 2658368
2 2 48
1 41 88 2710374
2 7 73
1 18 20 2935553
2 6 47
1 13 59 1567563
2 36 39
1 6 34 1792484
2 3 79
1 23 97 1314947
2 2 70
1 11 21 2368877
2 11 23
1 24 75 2285508
2 21 32
1 80 96 2789047
2 38 59
1 14 15 146611
2 18 41
1 15 87 2611104
2 64 92
1 38 75 211277
2 43 88
