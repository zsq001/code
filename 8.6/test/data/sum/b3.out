998320276
640036978
537573687
930418833
462280305
858142803
639686087
975173644
233719833
644978147
629540948
43043989
577258324
361547537
592211005
872933666
512264495
332947864
385682330
470207145
662419694
356015013
666225358
706222566
956274705
815789299
602487411
818330108
651493051
857497922
389278058
858999310
443841124
392616629
647932989
561975785
116946686
743936989
99110836
466708466
97410517
451744421
705680380
794844525
607800659
424802218
361083803
313142588
584270297
873063327
